management/src/main/java/org/kaazing/gateway/management/monitoring/writer/impl/MMFSeviceWriter.java

```diff
@@ -0,0 +1,71 @@
+/**
```

Comment:
Typo: the name of this class should be MMFServiceWriter (by the way, I expect you already know this, but just in case, it's really easy to rename methods or classes in Eclipse by right clicking on the name and choosing menu option Refactor).